### PR TITLE
Core-AAM: Fix CFRange property names for AXAPI

### PR DESCRIPTION
**Pull request by Joanmarie**
AXAPI exposes aria-colspan and aria-rowspan through AXColumnIndexRange
and AXRowIndexRange respectively. These range values are of type
kAXValueCFRangeType which is a wrapper for CFRange [1]. Looking at the
documentation for CFRange [2], the properties are location and length;
not loc and len.

[1] https://developer.apple.com/reference/applicationservices/axvaluetype/kaxvaluecfrangetype
[2] https://developer.apple.com/reference/corefoundation/cfrange

### CODE REVIEW

**Comment by klown** [comment]
@Joanmarie I've merged in your changes:
https://github.com/w3c/aria/commit/6ef5fba702115141cf48ff1e0ec71922f41155e6

rawgit:

- https://rawgit.com/w3c/aria/b68faca5c2e5a69ab0bf9715d3327008e9bab265/core-aam/core-aam.html#ariaRowSpan
- https://rawgit.com/w3c/aria/b68faca5c2e5a69ab0bf9715d3327008e9bab265/core-aam/core-aam.html#ariaColSpan

Feel free to close, or let me know if there are issues.

**Comment by Joanmarie** [comment]
@klown regarding this:

> The "len" and "loc" forms were confirmed in [gh-issue-459](https://github.com/w3c/aria/issues/459#issuecomment-272947803). Did the mappings change? Or was the documentation wrong back then?

TL;DR: It was more a case of not finding the documentation and believing tools like the Accessibility Inspector and other specs like HTML-AAM over the code. (In retrospect, that was extremely silly of me.)

Longer answer:  If you look at the comment you cite, you'll note I stated the following:

    And if you look at the associated code, you'll see the values come
    from AXRowIndexRange and AXColumIndexRange respectively. And for
    each of those, the value is a (range.location, range.length) pair.

So the code I reference shows that the mapping has the fully-written-out names. But then I said:

    ... Mind you, I was a tad surprised by the abbreviations "loc" and
    "len" because that seems really unWebKitish. But I just built Safari,
    loaded the test case, and examined it using Accessibility Inspector.
    It is indeed "loc" and "len". See attached screenshot... Lastly, all
    this jives with what's in the HTML-AAM.

Apologies for not having poked around more for the API docs which document what I found in the code.
